server/app/src/test/java/com/google/heartrate/wearos/app/gatt/TestUtils.java

```diff
@@ -0,0 +1,25 @@
+package com.google.heartrate.wearos.app.gatt;
+
+import org.junit.function.ThrowingRunnable;
+
+import static org.junit.Assert.fail;
+
+/**
+ * Class TestUtils provide useful methods and custom assertions for testing.
+ */
+public class TestUtils {
+
+    /**
+     * Assert given throwing runnable arguments does not throw exception.
+     * <br>Fail on runnable arguments throw exception on run().
+     *
+     * @param throwingRunnable throwing runnable arguments to check
+     */
+    public static void assertNotThrows(ThrowingRunnable throwingRunnable) {
```

Comment:
I think this is the default behavior of tests: when something does not throw the test will not fail, if it throws it will fail. Try it out, I don't think you need this. You may however need a assertThrows method.

Reply:
Before implementing testing methods yourself I'd consider using https://github.com/google/truth, which is a very helpful library for writing tests.

Reply:
I was not right using such method. Removed it.